packages/card-drawer/src/CardDrawer.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import Collapsible from 'react-collapsible'
+
+// hixme-ui
+import Container from '@hixme-ui/container'
+import Icon from '@hixme-ui/icon'
+
+// Styled components
+import ChevronRow from './ChevronRow'
+import Disabled from './Disabled'
+import InnerContent from './InnerContent'
+import StyledCard from './StyledCard'
+import StyledTrigger from './StyledTrigger'
+
+// Collapsible class names
+const ContentInnerClassName = 'Collapsible__contentInner'
+const TriggerClassName = 'Collapsible__trigger'
+
+class CardDrawer extends Component {
+  static propTypes = {
+    cardContent: PropTypes.node,
+    cardHeader: PropTypes.node,
+    children: PropTypes.node,
+    hideArrow: PropTypes.bool,
+    collapsible: PropTypes.bool,
+    open: PropTypes.bool,
+    disabled: PropTypes.bool,
+  }
+
+  static defaultProps = {
+    cardContent: null,
+    cardHeader: null,
+    children: null,
+    hideArrow: false,
+    collapsible: false,
+    open: true,
+    disabled: false,
+  }
+
+  constructor(props) {
```

Comment:
Nothing wrong with this but just wanted to let you know you do not need to declare a constructor to initialize state. [Here is a good article about that](https://michalzalecki.com/react-components-and-class-properties/).

```
class MyComponent extends Component {
  state = { open: props.open }

  ... the rest of my code with no constructor ...
}
```

Reply:
Didn't know that. Thanks!